src/librustc_mir/transform/mod.rs

```diff
@@ -311,12 +311,12 @@ fn run_optimization_passes<'tcx>(
             &const_prop::ConstProp,
             &simplify_branches::SimplifyBranches::new("after-const-prop"),
             &deaggregator::Deaggregator,
+            &simplify_try::SimplifyArmIdentity,
+            &simplify_try::SimplifyBranchSame,
             &copy_prop::CopyPropagation,
             &simplify_branches::SimplifyBranches::new("after-copy-prop"),
             &remove_noop_landing_pads::RemoveNoopLandingPads,
             &simplify::SimplifyCfg::new("after-remove-noop-landing-pads"),
```

Comment:
This simplifycfg is now redundant

Reply:
Why? I thought this was tied to the `RemoveNoopLandingPads` pass which is unchanged.

Reply:
`SimplifyCfg::new("after-remove-noop-landing-pads")` and `SimplifyCfg::new("final")` run consecutively after this change.

```diff
             &simplify::SimplifyCfg::new("after-remove-noop-landing-pads"),
-            &simplify_try::SimplifyArmIdentity,
-            &simplify_try::SimplifyBranchSame,
             &simplify::SimplifyCfg::new("final"),
```

Reply:
Got it! The GitHub comment view was not showing that line.